src/devices/sound/k054539.cpp

```diff
@@ -104,13 +104,34 @@ void k054539_device::keyoff(int channel)
 		regs[0x22c] &= ~(1 << channel);
 }
 
+void k054539_device::advance_filter(int channel, int val)
+{
+	float *hist = filter_hist[channel];
```

Comment:
The pointer here can be `const` since you don’t plan to change – you could do `float *const hist = filter_hist[channel];` (or maybe even use a reference like `auto &hist = filter_hist[channel];`)